.mise/tasks/platform/dataprovider/build

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env sh
+
+# mise description="build docker image for the data provider"
+# mise tools=["poetry"]
+
+cd $root/platform/dataprovider
+
+IMAGE=pocketsizefund/data-provider
+VERSION=$(poetry version -s)
```

Comment:
use `cargo pkgid | awk -F# '{print $NF}'` for cargo

Reply:
I reliaze that's kinda ugly, I can't find something more elegant built into cargo